app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+class Animal:
+    hungry_animals = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        if is_hungry:
```

Comment:
You do not need to store hungry animals in separeted list . You just can use the `is_hungry` attribute